CHANGELOG.md

```diff
@@ -5,11 +5,25 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
 ## [Unreleased]
+## [0.5.0] - 2015-12-22
+### Changed
+- updated the way makefiles are generated
```

Comment:
I would probably write this out a little more. I'll give you some copy to think about in a minute.